test/dpe_verification/dpe_test.go

```diff
@@ -3,13 +3,15 @@
 package dpe
 
 import (
-	"github.com/chipsalliance/caliptra-dpe/verification"
 	"os"
 	"testing"
+
+	"github.com/chipsalliance/caliptra-dpe/verification/client"
+	DpeTesting "github.com/chipsalliance/caliptra-dpe/verification/testing"
```

Comment:
nit: Generally golang packages are all lower case. `dpetesting` would probably be the idiomatic option

Reply:
Done